pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -49,10 +50,11 @@ func (r *Revision) checkImmutableFields(ctx context.Context, original *Revision)
 // Validate ensures Revision is properly configured.
 func (r *Revision) Validate(ctx context.Context) *apis.FieldError {
```

Comment:
any changes to v1alpha1 should be reflected in v1beta1, which we will start using imminently.

Reply:
ok good, you do :)